translib/ocbinds/schema.go

```diff
@@ -0,0 +1,34 @@
+////////////////////////////////////////////////////////////////////////////////
+//                                                                            //
+//  Copyright 2021 Broadcom. The term Broadcom refers to Broadcom Inc. and/or //
+//  its subsidiaries.                                                         //
+//                                                                            //
+//  Licensed under the Apache License, Version 2.0 (the "License");           //
+//  you may not use this file except in compliance with the License.          //
+//  You may obtain a copy of the License at                                   //
+//                                                                            //
+//     http://www.apache.org/licenses/LICENSE-2.0                             //
+//                                                                            //
+//  Unless required by applicable law or agreed to in writing, software       //
+//  distributed under the License is distributed on an "AS IS" BASIS,         //
+//  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.  //
+//  See the License for the specific language governing permissions and       //
+//  limitations under the License.                                            //
+//                                                                            //
+////////////////////////////////////////////////////////////////////////////////
+
+package ocbinds
+
+import (
+	"github.com/openconfig/ygot/ytypes"
+)
+
+// GetSchema is equivalent of the Schema function, but avoids
+// UnzipSchema call. Reuses the SchemaTree loaded during init.
+func GetSchema() (*ytypes.Schema, error) {
+	return &ytypes.Schema{
+		Root:       &Device{},
+		SchemaTree: SchemaTree,
+		Unmarshal:  Unmarshal,
+	}, nil
```

Comment:
GetSchema() only return nil as error. Consider removing error from output.

Reply:
I retained the signature of the ygot generated `Schema()` function